src/Feature/Editor/EditorSurface.re

```diff
@@ -113,6 +113,9 @@ let minimap =
 
 let make =
     (
+      ~showDiffMarkers=true,
+      ~backgroundColor: option(Revery.Color.t)=?,
+      ~foregroundColor: option(Revery.Color.t)=?,
```

Comment:
This might be a bit confusing - these optional parameters override some of the configuration / theme settings that come in. @glennsl  - are you OK with this model? Or do you have another approach you'd prefer?

Reply:
I don't see any major issues with it, so I think it's fine. Clever use of the precomputed colors too. This should also work fine after extracting the editor as an app-independent component I think, once we get around to doing that.